Formula/m/mono.rb

```diff
@@ -1,8 +1,9 @@
 class Mono < Formula
   desc "Cross platform, open source .NET development framework"
   homepage "https://www.mono-project.com/"
-  url "https://download.mono-project.com/sources/mono/mono-6.12.0.199.tar.xz"
-  sha256 "c0850d545353a6ba2238d45f0914490c6a14a0017f151d3905b558f033478ef5"
+  url "https://github.com/mono/mono.git",
+      tag:      "mono-6.12.0.206",
+      revision: "0cbf0e290c31adb476f9de0fa44b1d8829affa40"
```

Comment:
Can we fetch the Git archive instead? A Git clone is usually much heavier and less cache-friendly.

```suggestion
  url "https://github.com/mono/mono/archive/refs/tags/mono-6.12.0.206.tar.gz"
  sha256 "52ca15df4e29d08fbeb7a85bb35db1b2313521999473c5960e50d9bd56389c2e"
```

Reply:
I already tried that but I couldn't clone the submodules in this way.